db/seeds/evaluatieonderzoek/one_time_reponse_seeds.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+OneTimeResponse.delete_all
```

Comment:
`.destroy_all`? Dan ruimt ie eventuele dependent objects netjes op volgens mij.

Reply:
Misschien willen we trouwens om dit hele blok een if development environemtn statement zetten, en dat we de token voor het evaluatieonderzoek in de console genereren. Dan heb je namelijk niet dat het token in de (publieke) github repo komt te staan en dat iedereen m dus kan zien.